docs/source/variables.rst

```diff
@@ -0,0 +1,48 @@
+Variables
+=========
+
+.. image:: _static/images/variables_initial.png
+
+This panel shows the variables available along with their properties.
+It also allows the user to select a variable for further analysis in
```

Comment:
You do not do analysis in the other panels. This is not really an *also*, it is the primary function of this tab.